irodsadmin/tests/test_commands.py

```diff
@@ -437,13 +463,17 @@ def test_command_orphan_assay_sub(self):
         )
         self.irods.collections.create(orphan_path)
         out = io.StringIO()
+        sys.stdout = out
         call_command('irodsorphans', stdout=out)
+        output = out.getvalue()
```

Comment:
The following bit seems to be repeated quite a lot in the tests without change:

```
out = io.StringIO()
sys.stdout = out
call_command('irodsorphans', stdout=out)
output = out.getvalue()
```

How about making this a protected helper method in the test class which returns `output`?